doc-Provisioning_Virtual_Machines_and_Hosts/topics/Provisioning_Requests.adoc

```diff
@@ -149,13 +149,23 @@ image:2320.png[]
 [[iso-provisioning]]
 === ISO Provisioning
 
-{product-title} also allows ISO provisioning from Red Hat Virtualization Manager datastores. To use this feature, you will need to do the following before creating a provision request.
+{product-title} also allows ISO provisioning from Red Hat Virtualization Manager datastores. 
+
+*Requirements for ISO Provisioning*
+
+* ISO provisioning requires a customization template to provide Kickstart files for initial uploading of the operating system. See xref:customization-templates-for-virtual-machine-and-instance-provisioning[] for more information on customization templates.
+* ISO Provisioning on Red Hat Virtualization will write the content generated by the customization template to a file named `ks.cfg` that will be attached to the virtual machine using the RHV floppy payload. If this content is to be included in the installation, the ISO image must be aware of this and include the contents of the file. This requires remastering of the ISO to include a custom Kickstart file. 
+* For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/installation_guide/sect-kickstart-howto[How Do You Perform a Kickstart Installation] in the Red Hat Enterprise Linux _Installation Guide_.
+
+
+==== Initial Workflow
+To use this feature, you will need to do the following before creating a provision request.
 
 . Add the *ISO Datastore*. The Red Hat Virtualization Manager system must have already been discovered or added into the VMDB. For more information, see https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7-Beta/html-single/managing_providers/#adding_a_red_hat_enterprise_virtualization_manager_provider[Adding a Red Hat Enterprise Virtualization Manager Provider] in _Managing Providers_.
 . Refresh the *ISO Datastore*.
```

Comment:
I believe that this would be great place to reference this: https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.2/html-single/administration_guide/#sect-The_ISO_Uploader_Tool

Reply:
hi @jan-zmeskal I've updated the doc with your suggestion here. Since the uploader tool is deprecated with RHV 4.2, I've added some bullet points directing users the section appropriate for their version of RHV.

Reply:
@cbudz Great, thanks! I had to interrupt my review yesterday in the middle, but I am gonna finish it today and provide my final feedback. Thank you for the patience.